core/config.go

```diff
@@ -31,7 +31,7 @@ import (
 const defaultPrefix = "NUTS_"
 const defaultDelimiter = "."
 
-func loadConfigIntoStruct(flags *pflag.FlagSet, target interface{}, configMap *koanf.Koanf) error {
+func loadConfigIntoStruct(flags *pflag.FlagSet, target interface{}, flat bool, configMap *koanf.Koanf) error {
```

Comment:
In this PR, only `false` is passed for `flat`.

Reply:
Yeah indeed, will fix